### PR TITLE
[#155622636] Add cookies page 

### PR DESCRIPTION
**Pull request by LeePorte**
## What

This PR adds in the new cookies page to the product pages in advance of GDPR coming into force.

## How to review

Code review

launch this branch in development mode locally with `DESKPRO_TEAM_ID=1 DESKPRO_API_KEY='REDACTED' DESKPRO_ENDPOINT='https://account.deskpro.com' make dev` and navigate to the page via the footer link

## Who can merge

Not @LeePorte

### CODE REVIEW

File: views/cookies.erb [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	<div class="grid-row">
+
+		<div class="column-two-thirds">
+			<h1 class="heading-large">Cookies</h1>
+			<p class="summary">GOV.UK PaaS puts small files (known as ‘cookies’) onto your computer.
+			</p>
+			<p>Cookies are used to:</p>
+			<ul class="list list-bullet">
+				<li>remember you once you've signed in</li>
+				<li>keep your session open</li>
```
Comment (samcrang) [inline views/cookies.erb:19]: Are we okay with the wording here? I appreciate that we don't want to confuse things by getting "too technical" but i feel like "keep your session open" is very hand-wavy.

File: views/cookies.erb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+			<ul class="list list-bullet">
+				<li>the pages you visit </li>
+				<li>how long you spend on each page
</li>
+				<li>how you arrived at the site 
</li>
+				<li>what you click on while you’re visiting the site</li>
+				<li>the device and browser you’re using</li>
+			</ul>
+			<p>We don’t collect or store your personal information (for example your name or address) so this information can’t be used to identify who you are.</p>
```
Comment (samcrang) [inline views/cookies.erb:35]: Is it true that we don't store enough information to practically deanonymize someone? I think we should make it clear that we don't store your name (presumably unless you're a tenant) or address but I'd suspect that it is practical to deanonymize from data points collected by Google Analytics.

File: views/layout.erb [hunk 3]
```diff
@@ -91,6 +91,12 @@
 
 			<div class="footer-meta">
 				<div class="footer-meta-inner">
+					<nav class="footer-nav">
+							  <ul>
+								<li>Built by the <a href="https://www.gov.uk/government/organisations/government-digital-service">Government Digital Service</a></li>
+								<li><a href="/cookies">Cookies</a></li>
+							  </ul>
+							</nav>
```
Comment (samcrang) [inline views/layout.erb]: It kind of upsets me that the tabs and spaces are wonky 😬

**Review by samcrang — CHANGES_REQUESTED** [review]

File: views/cookies.erb (hunk 1 above)
Comment (LeePorte) [reply views/cookies.erb:19]: It's the wording we have been provided that has gone through IA.

Comment (samcrang) [reply views/cookies.erb:19]: :ok:

File: views/cookies.erb (hunk 2 above)
Comment (LeePorte) [reply views/cookies.erb:35]: The GaaP team have done work to anonymise what is collected by GA. We certainly don't have IP addresses or any other GDPR data collected by it

File: views/layout.erb (hunk 3 above)
Comment (LeePorte) [reply views/layout.erb]: I'll fix that

File: views/cookies.erb (hunk 2 above)
Comment (samcrang) [reply views/cookies.erb:35]: 🆗

File: views/layout.erb (hunk 3 above)
Comment (samcrang) [reply views/layout.erb]: 🆗

**Review by samcrang — APPROVED** [review]

**Pull request merged**